backend/tests/test_tenant_config_model.py

```
@@ -28,7 +28,13 @@ def test_tenant_config_model_exists():
 
 
 def test_tenant_config_model_encryption(db_session):
-    config = TenantConfig(user_id="test_user2", openai_api_key="test_key2")
+    config = TenantConfig(
+        user_id="test_user2",
+        openai_api_key="test_key2",
+        imap_username="mail-user",
+        imap_password="imap-secret",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "imap_password"

(S106)